spec/unit/helpers_spec.rb

```diff
@@ -0,0 +1,81 @@
+# Author:: Kartik Cating-Subramanian (<ksubramanian@chef.io>)
+# Copyright:: Copyright (c) 2015 Chef, Inc.
+# License:: Apache License, Version 2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'spec_helper'
+require_relative 'test_handler'
+
+describe ChefHandler::Helpers do
+  before do
+    extend ChefHandler::Helpers
```

Comment:
You sometimes see this as a pattern with ChefSpec tests when the libraries aren't loaded until the converge, but you are side-stepping that problem and just loading manually rather than via ChefSpec.